Source/WebCore/css/CSSStyleDeclaration.h

```diff
@@ -36,12 +37,10 @@ class MutableStyleProperties;
 class StyleProperties;
 class StyledElement;
 
-class CSSStyleDeclaration : public ScriptWrappable, public CanMakeSingleThreadWeakPtr<CSSStyleDeclaration> {
+class CSSStyleDeclaration : public AbstractRefCountedAndCanMakeSingleThreadWeakPtr<CSSStyleDeclaration>, public ScriptWrappable {
```

Comment:
A little worried about swapping the order of base classes here. Having `ScriptWrappable` as first base class made more sense to me (though hopefully the bindings code doesn't rely on that?)